hack/update-codegen.sh

```diff
@@ -42,15 +42,17 @@ EXTERNAL_INFORMER_PKG="k8s.io/client-go/informers" \
     k8s.io/client-go \
     k8s.io/api \
     "admissionregistration:v1beta1 apps:v1 autoscaling:v1,v2beta1 batch:v1,v1beta1 core:v1 rbac:v1" \
-    --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
+    --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt \
+    --force-kinds "Namespace"
```

Comment:
Can we `s/kinds/genreconciler/` for clarity?

Reply:
sure